+ JOB=3
+ NO_JOBS=16
+ [[ ! -n 3 ]]
+ [[ ! -n 16 ]]
+ [[ 3 == global ]]
+ set +x
+ pkgcheck -r gentoo --reporter FancyReporter 'dev-haskell/*' 'net-analyzer/*' 'app-vim/*' 'app-xemacs/*' 'games-arcade/*' 'dev-cpp/*' 'sci-physics/*' 'net-nds/*' 'dev-lua/*' 'dev-go/*' -d imlate -d unstable_only -d cleanup -d stale_unstable -d UnusedGlobalFlags -d UnusedLicense --profile-disable-dev --profile-disable-exp
WARNING pkgcore: invalid glsa- glsa-201206-24.xml, package www-servers/tomcat:
               : error range rlt version 5.5.34 is a guaranteed empty set

dev-haskell/alut
  DeprecatedEAPI: version 2.1.0.0: uses deprecated EAPI: 0

dev-haskell/bifunctors
  CrappyDescription: version 4.1.0.1: description needs improvement: using the pkg name as the description isn't very helpful
  CrappyDescription: version 4.1.1: description needs improvement: using the pkg name as the description isn't very helpful
  CrappyDescription: version 4.1.1.1: description needs improvement: using the pkg name as the description isn't very helpful
  CrappyDescription: version 4.2: description needs improvement: using the pkg name as the description isn't very helpful
  CrappyDescription: version 4.2.1: description needs improvement: using the pkg name as the description isn't very helpful

dev-haskell/c2hs
  DroppedKeywordWarning: version 0.17.2: keywords dropped: alpha, ia64, ppc, ppc64, sparc

dev-haskell/cairo
  DeprecatedEAPI: version 0.12.0: uses deprecated EAPI: 3

dev-haskell/cgi
  DroppedKeywordWarning: version 3001.2.2.0: keywords dropped: alpha, ppc, sparc

dev-haskell/convertible
  DeprecatedEAPI: version 1.0.10.0: uses deprecated EAPI: 3

dev-haskell/cpphs
  DroppedKeywordWarning: version 1.18.5: keywords dropped: alpha

dev-haskell/data-binary-ieee754
  DeprecatedEAPI: version 0.4.2.1: uses deprecated EAPI: 3

dev-haskell/dbus
  DroppedKeywordWarning: version 0.10.9.2: keywords dropped: alpha, ia64, ppc, ppc64, sparc

dev-haskell/dlist
  DeprecatedEAPI: version 0.5: uses deprecated EAPI: 3

dev-haskell/drift
  DeprecatedEAPI: version 2.2.0: uses deprecated EAPI: 0
  DroppedKeywordWarning: version 2.4.2: keywords dropped: ia64

dev-haskell/erf
  DeprecatedEAPI: version 2.0.0.0: uses deprecated EAPI: 3

dev-haskell/gio
  DeprecatedEAPI: version 0.12.0: uses deprecated EAPI: 3

dev-haskell/glib
  DeprecatedEAPI: version 0.12.0: uses deprecated EAPI: 3
  DroppedKeywordWarning: version 0.13.0.6: keywords dropped: ia64

dev-haskell/glut
  DeprecatedEAPI: version 2.1.1.1: uses deprecated EAPI: 0
  DeprecatedEAPI: version 2.1.1.2: uses deprecated EAPI: 0

dev-haskell/gtk
  DeprecatedEAPI: version 0.12.0: uses deprecated EAPI: 3

dev-haskell/haddock
  DroppedKeywordWarning: version 2.13.1-r2: keywords dropped: alpha, ia64, ppc, ppc64, sparc

dev-haskell/harp
  DeprecatedEAPI: version 0.4: uses deprecated EAPI: 3

dev-haskell/haskell-src
  DeprecatedEAPI: version 1.0.1.4: uses deprecated EAPI: 3
  DeprecatedEAPI: version 1.0.1.5: uses deprecated EAPI: 3

dev-haskell/hpc
  DeprecatedEAPI: version 0.5.1.1: uses deprecated EAPI: 3

dev-haskell/hscolour
  DeprecatedEAPI: version 1.17: uses deprecated EAPI: 0

dev-haskell/hsql-mysql
  DroppedKeywordWarning: version 1.8.3: keywords dropped: amd64, ppc, ppc64, sparc, x86

dev-haskell/html
  DeprecatedEAPI: version 1.0.1.2: uses deprecated EAPI: 0

dev-haskell/http
  DeprecatedEAPI: version 4000.1.1: uses deprecated EAPI: 3
  DroppedKeywordWarning: version 4000.2.18: keywords dropped: alpha, ia64, ppc, ppc64, sparc

dev-haskell/hunit
  DeprecatedEAPI: version 1.2.4.2: uses deprecated EAPI: 3

dev-haskell/hxt-charproperties
  DeprecatedEAPI: version 9.1.0: uses deprecated EAPI: 3

dev-haskell/json
  DeprecatedEAPI: version 0.4.3: uses deprecated EAPI: 0
  DeprecatedEAPI: version 0.4.4: uses deprecated EAPI: 3
  DeprecatedEAPI: version 0.5: uses deprecated EAPI: 3

dev-haskell/libxml-sax
  DeprecatedEAPI: version 0.7.2: uses deprecated EAPI: 3

dev-haskell/maccatcher
  DeprecatedEAPI: version 2.1.5: uses deprecated EAPI: 3

dev-haskell/mersenne-random-pure64
  DeprecatedEAPI: version 0.2.0.3: uses deprecated EAPI: 3

dev-haskell/monad-control
  DeprecatedEAPI: version 0.2.0.3: uses deprecated EAPI: 3

dev-haskell/monadcatchio-mtl
  DroppedKeywordWarning: version 0.3.1.0-r1: keywords dropped: alpha, ppc, sparc

dev-haskell/mtl
  DeprecatedEAPI: version 2.0.1.0: uses deprecated EAPI: 3

dev-haskell/openal
  DeprecatedEAPI: version 1.3.1.1: uses deprecated EAPI: 0

dev-haskell/opengl
  DeprecatedEAPI: version 2.2.1.1: uses deprecated EAPI: 0

dev-haskell/pango
  DeprecatedEAPI: version 0.12.0: uses deprecated EAPI: 3

dev-haskell/parallel
  DeprecatedEAPI: version 1.0.0.0: uses deprecated EAPI: 0
  DeprecatedEAPI: version 1.1.0.1: uses deprecated EAPI: 0
  DeprecatedEAPI: version 3.2.0.2: uses deprecated EAPI: 3

dev-haskell/parsec
  DeprecatedEAPI: version 2.1.0.1: uses deprecated EAPI: 0

dev-haskell/pcap
  DeprecatedEAPI: version 0.4.5.1: uses deprecated EAPI: 3

dev-haskell/profunctors
  CrappyDescription: version 4.0.1: description needs improvement: using the pkg name as the description isn't very helpful
  CrappyDescription: version 4.0.4: description needs improvement: using the pkg name as the description isn't very helpful
  CrappyDescription: version 4.3.2: description needs improvement: using the pkg name as the description isn't very helpful
  CrappyDescription: version 4.4.1: description needs improvement: using the pkg name as the description isn't very helpful

dev-haskell/quickcheck
  DeprecatedEAPI: version 1.2.0.1: uses deprecated EAPI: 0
  DroppedKeywordWarning: version 2.7.6: keywords dropped: alpha, ia64, ppc, ppc64, sparc

dev-haskell/ranges
  DeprecatedEAPI: version 0.2.4: uses deprecated EAPI: 3

dev-haskell/regex-compat
  DeprecatedEAPI: version 0.95.1: uses deprecated EAPI: 3

dev-haskell/regex-tdfa
  DeprecatedEAPI: version 1.1.8: uses deprecated EAPI: 3

dev-haskell/sha
  DeprecatedEAPI: version 1.5.0.0: uses deprecated EAPI: 3

dev-haskell/statevar
  DeprecatedEAPI: version 1.0.0.0: uses deprecated EAPI: 3

dev-haskell/stm
  DeprecatedEAPI: version 2.1.2.2: uses deprecated EAPI: 0

dev-haskell/syb
  DeprecatedEAPI: version 0.3.2: uses deprecated EAPI: 3
  DeprecatedEAPI: version 0.3.6: uses deprecated EAPI: 3

dev-haskell/tar
  DeprecatedEAPI: version 0.3.1.0: uses deprecated EAPI: 3
  DeprecatedEAPI: version 0.3.2.0: uses deprecated EAPI: 3

dev-haskell/tasty
  DroppedKeywordWarning: version 0.7: keywords dropped: ppc, ppc64, sparc

dev-haskell/tasty-hunit
  DroppedKeywordWarning: version 0.8.0.1: keywords dropped: ppc, ppc64, sparc

dev-haskell/tasty-smallcheck
  DroppedKeywordWarning: version 0.8: keywords dropped: ppc, ppc64, sparc

dev-haskell/terminfo
  DeprecatedEAPI: version 0.3.2.3: uses deprecated EAPI: 3

dev-haskell/test-framework
  DeprecatedEAPI: version 0.2.4: uses deprecated EAPI: 0
  DeprecatedEAPI: version 0.4.2.0: uses deprecated EAPI: 3
  DroppedKeywordWarning: version 0.4.2.0: keywords dropped: alpha, ia64, ppc, sparc

dev-haskell/test-framework-hunit
  DeprecatedEAPI: version 0.2.4: uses deprecated EAPI: 0
  DeprecatedEAPI: version 0.2.6: uses deprecated EAPI: 3

dev-haskell/testpack
  DeprecatedEAPI: version 2.1.1: uses deprecated EAPI: 3

dev-haskell/transformers
  DeprecatedEAPI: version 0.2.2.0: uses deprecated EAPI: 3

dev-haskell/uuagc
  DeprecatedEAPI: version 0.9.24: uses deprecated EAPI: 0

dev-haskell/uulib
  DeprecatedEAPI: version 0.9.14: uses deprecated EAPI: 3

dev-haskell/wxhaskell
  CrappyDescription: version 0.13.2.3: description needs improvement: using the pkg name as the description isn't very helpful
  CrappyDescription: version 0.90.1.0: description needs improvement: using the pkg name as the description isn't very helpful

dev-haskell/xml
  DeprecatedEAPI: version 1.3.8: uses deprecated EAPI: 3

dev-haskell/xml-types
  DeprecatedEAPI: version 0.3: uses deprecated EAPI: 3
WARNING pkgcore: invalid glsa- glsa-201206-24.xml, package www-servers/tomcat:
               : error range rlt version 5.5.34 is a guaranteed empty set

net-analyzer/2ping
  MissingChksum: version 1.2.3: "2ping-1.2.3.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : dev-libs/cyrus-sasl-2.1.26-r9, exception Metadata Exception:
             : pkg ebuild src: dev-libs/cyrus-sasl-2.1.26-r9, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : dev-libs/cyrus-sasl-2.1.26-r9, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fcad290c350> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : dev-libs/cyrus-sasl-2.1.26-r9, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : dev-libs/cyrus-sasl-2.1.26-r9, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : dev-libs/cyrus-sasl-2.1.26-r9, exception Metadata Exception:
             : pkg ebuild src: dev-libs/cyrus-sasl-2.1.26-r9, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : dev-libs/cyrus-sasl-2.1.26-r9, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fcad290c350> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : dev-libs/cyrus-sasl-2.1.26-r9, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : dev-libs/cyrus-sasl-2.1.26-r9, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : dev-libs/cyrus-sasl-2.1.26-r9, exception Metadata Exception:
             : pkg ebuild src: dev-libs/cyrus-sasl-2.1.26-r9, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : dev-libs/cyrus-sasl-2.1.26-r9, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fcad290c350> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : dev-libs/cyrus-sasl-2.1.26-r9, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : dev-libs/cyrus-sasl-2.1.26-r9, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fcad290c350> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'

net-analyzer/authforce
  MissingChksum: version 0.9.9-r1: "authforce-0.9.9.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-analyzer/barnyard
  DeprecatedEAPI: version 0.2.0-r3: uses deprecated EAPI: 2
  MissingChksum: version 0.2.0-r3: "barnyard-0.2.0.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 0.2.0-r3: "barnyard-0.2.0-patches.tar.bz2" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-analyzer/barnyard2
  DeprecatedEAPI: version 1.9: uses deprecated EAPI: 2
  MissingChksum: version 1.9: "barnyard2-1.9.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-analyzer/bing
  DeprecatedEAPI: version 1.1.3-r1: uses deprecated EAPI: 2
  MissingChksum: version 1.1.3-r1: "bing_1.1.3.orig.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-analyzer/bsnmp
  DeprecatedEAPI: version 1.11a: uses deprecated EAPI: 0
  DeprecatedEAPI: version 1.12: uses deprecated EAPI: 0
  DeprecatedEAPI: version 1.12-r1: uses deprecated EAPI: 0
  MissingChksum: version 1.11a: "bsnmp-1.11a.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-analyzer/chaosreader
  DeprecatedEAPI: version 0.94: uses deprecated EAPI: 2
  MissingChksum: version 0.94: "chaosreader-0.94.bz2" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-analyzer/dhcp_probe
  DeprecatedEAPI: version 1.3.0-r3: uses deprecated EAPI: 2
  MissingChksum: version 1.3.0-r3: "dhcp_probe-1.3.0.tar.gz" missing required chksums: sha512, whirlpool; has chksums: rmd160, sha1, sha256, size

net-analyzer/dietsniff
  MissingChksum: version 0.4: "dietsniff-0.4.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-analyzer/dsniff
  DeprecatedEAPI: version 2.4_beta1-r4: uses deprecated EAPI: 0

net-analyzer/echoping
  VulnerablePackage: version 6.0.2-r2: vulnerable via glsa(201406-07) ( ver < 6.0.2_p434 ), keywords ('amd64', 'hppa', 'x86')
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fcad290c350> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'

net-analyzer/flow-tools
  ExecutableFile: "files/flowcapture.confd" has unnecessary executable bit
  ExecutableFile: "files/flowcapture.initd" has unnecessary executable bit
  ExecutableFile: "files/linkme" has unnecessary executable bit

net-analyzer/fprobe
  DeprecatedEAPI: version 1.1: uses deprecated EAPI: 0
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fcad290c350> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fcad290c350> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'

net-analyzer/gr-fosphor
  DeprecatedEclass: version 9999: uses deprecated eclass(es): [ git-2 (migrate to git-r3) ]

net-analyzer/greenbone-security-assistant
  ExecutableFile: "files/gsad" has unnecessary executable bit

net-analyzer/gsm-receiver
  DeprecatedEclass: version 9999: uses deprecated eclass(es): [ git-2 (migrate to git-r3) ]

net-analyzer/gspoof
  MissingChksum: version 3.2-r2: "gspoof-3.2.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-analyzer/hping
  DeprecatedEAPI: version 3_pre20051105-r3: uses deprecated EAPI: 2

net-analyzer/icinga2
  DeprecatedEclass: version 9999: uses deprecated eclass(es): [ git-2 (migrate to git-r3) ]

net-analyzer/ike-scan
  MissingChksum: version 1.9-r1: "ike-scan-1.9.tar.gz" missing required chksums: sha512, whirlpool; has chksums: rmd160, sha1, sha256, size

net-analyzer/ipaudit
  DeprecatedEAPI: version 1.0_beta2: uses deprecated EAPI: 0
  MissingChksum: version 1.0_beta2: "ipaudit-1.0BETA2.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fcad290c350> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'

net-analyzer/metasploit
  DeprecatedEAPI: version 2.7: uses deprecated EAPI: 0
  DroppedKeywordWarning: version 4.9.3: keywords dropped: ppc

net-analyzer/multimon-ng
  DeprecatedEclass: version 9999: uses deprecated eclass(es): [ git-2 (migrate to git-r3) ]

net-analyzer/munin
  DeprecatedEclass: version 2.1.9999: uses deprecated eclass(es): [ git-2 (migrate to git-r3) ]

net-analyzer/nagios-check_logfiles
  DeprecatedEAPI: version 2.3.2.1-r1: uses deprecated EAPI: 0

net-analyzer/nagios-check_rbl
  DroppedKeywordWarning: version 1.3.7: keywords dropped: x86

net-analyzer/nagios-plugins-flameeyes
  DeprecatedEclass: version 9999: uses deprecated eclass(es): [ git-2 (migrate to git-r3) ]

net-analyzer/nagios-sap-ccms-plugin
  DeprecatedEAPI: version 0.7.3-r1: uses deprecated EAPI: 0
  MissingChksum: version 0.7.3-r1: "sap-ccms-plugin-0.7.3.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 0.8: "sap-ccms-plugin-0.8.tar.bz2" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-analyzer/nagircbot
  ExecutableFile: "files/init" has unnecessary executable bit

net-analyzer/nagstamon
  DeprecatedEclass: version 0.9.11_rc1: uses deprecated eclass(es): [ python (migrate to python-r1 / python-single-r1 / python-any-r1) ]

net-analyzer/nagtrap
  DeprecatedEAPI: version 0.1.3: uses deprecated EAPI: 2

net-analyzer/ncrack
  DeprecatedEAPI: version 0.3_alpha: uses deprecated EAPI: 2
  DeprecatedEAPI: version 0.4_alpha: uses deprecated EAPI: 2
  MissingChksum: version 0.3_alpha: "ncrack-0.3ALPHA.tar.gz" missing required chksums: sha512, whirlpool; has chksums: rmd160, sha1, sha256, size
  MissingChksum: version 0.4_alpha: "ncrack-0.4ALPHA.tar.gz" missing required chksums: sha512, whirlpool; has chksums: rmd160, sha1, sha256, size

net-analyzer/ndoutils
  ExecutableFile: "files/ndo2db.init-nagios3" has unnecessary executable bit

net-analyzer/nepenthes
  DeprecatedEAPI: version 0.2.2: uses deprecated EAPI: 2

net-analyzer/nessus
  DeprecatedEAPI: version 2.2.9: uses deprecated EAPI: 0

net-analyzer/nessus-bin
  DeprecatedEAPI: version 4.4.1: uses deprecated EAPI: 0
  DeprecatedEAPI: version 5.0.1: uses deprecated EAPI: 0
  MissingChksum: version 4.4.1: "Nessus-4.4.1-es6.i686.rpm" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 4.4.1: "Nessus-4.4.1-es6.x86_64.rpm" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-analyzer/nessus-client
  DeprecatedEAPI: version 1.0.2-r1: uses deprecated EAPI: 2
  MissingChksum: version 1.0.2-r1: "NessusClient-1.0.2.tar.gz" missing required chksums: sha512, whirlpool; has chksums: rmd160, sha1, sha256, size
  MissingChksum: version 1.0.2-r1: "nessus-client.png.bz2" missing required chksums: sha512, whirlpool; has chksums: rmd160, sha1, sha256, size

net-analyzer/nessus-core
  NonExistentDeps: version 2.2.9-r1: depset depends: nonexistent atoms [ dev-libs/libprelude ]
  NonExistentDeps: version 2.2.9-r1: depset rdepends: nonexistent atoms [ dev-libs/libprelude ]
  MissingChksum: version 2.2.9-r1: "nessus-core-2.2.9.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-analyzer/nessus-libraries
  MissingChksum: version 2.2.9-r1: "nessus-libraries-2.2.9.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-analyzer/nessus-plugins
  MissingChksum: version 2.2.9-r1: "nessus-plugins-GPL-2.2.9.tar.gz" missing required chksums: sha512, whirlpool; has chksums: rmd160, sha1, sha256, size

net-analyzer/neti
  DeprecatedEAPI: version 2.0: uses deprecated EAPI: 0

net-analyzer/netsniff-ng
  DeprecatedEclass: version 9999: uses deprecated eclass(es): [ git-2 (migrate to git-r3) ]

net-analyzer/nikto
  DeprecatedEAPI: version 1.36: uses deprecated EAPI: 0
  MissingChksum: version 1.36: "nikto-1.36.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fcad290c350> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fcad290c350> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fcad290c350> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7fcad290c350> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(se